librz/main/rz-asm.c

```diff
@@ -217,7 +218,54 @@ static int rasm_show_help(int v) {
 	return 0;
 }
 
-static int rasm_disasm(RzAsmState *as, ut64 addr, const char *buf, int len, int bits, int bin, int hex) {
+typedef enum {
+	DISASM_MODE_DONT = 0,
+	DISASM_MODE_DEFAULT,
+	DISASM_MODE_WITH_BYTES,
+	DISASM_MODE_ESIL,
```

Comment:
Maybe move it to the end and name like `DISASM_MODE_ESIL_DEPRECATED` or something?

Reply:
I wouldn't consider this option deprecated as long as esil is still being used for different things.